poule.yml

```diff
@@ -33,13 +33,6 @@
                 platform/arm:        [ "raspberry", "raspbian", "rpi", "beaglebone", "pine64" ],
             }
         }
-      - type:       version-label
```

Comment:
in the near future, ppl should probably not report issues in moby/moby, but I'll add back for now